templates/admin.tpl.php

```diff
@@ -0,0 +1,127 @@
+<?php
+$module_path = backdrop_get_path('module', 'tawk_to');
+backdrop_add_css($module_path . '/css/tawk_to.admin.css', 'file');
+?>
+
+<div id="content" >
+  <div class="l-wrapper">
+    <div class="l-wrapper-inner container container-fluid">
+      <div class="l-messages success-message-container" role="status" aria-label="Status messages" id="optionsSuccessMessage">
+        <div class="messages status">
+          <h2 class="element-invisible">Status message</h2>
+          Successfully set widget options to your site
+        </div>
+      </div>
+      <div class="l-page-title">
+        <a id="main-content"></a>
+      </div>
+      <div class="l-content" role="main" aria-label="Main content">
+        <div>
+          <fieldset class="form-wrapper" id="property-widget-settings">
+            <legend><span class="fieldset-legend">Property and Widget Selection</span></legend>
+            <div class="fieldset-wrapper">
+              <div class="fieldset-description">Select the property and widget from your tawk.to account</div>
+              <?php if (!$sameUser) { ?>
+                <div id="widget_already_set" class="alert alert-warning">Notice: Widget already set by other user</div>
+              <?php } ?>
+              <iframe id="tawkIframe" class="tawkto-property-widget-selection" src="" ></iframe>
+              <input type="hidden" class="hidden" name="page_id" value="<?php echo $widget['page_id']?>">
```

Comment:
this fieldset should also be added to the page via the formAPI. 

You can do something like this:
```php
global $user;
  $config = config('tawk_to.settings');
  $admin_uid = $config->get('uid');
  if (isset($admin_uid) && $admin_uid != $user->uid) {
    $message = t('Notice: The widget has already been set by abother person.');
    backdrop_set_message($message, 'warning');
  }

  $form['widget'] = array(
    '#type' => 'fieldset',
    '#title' => t('Property and Widget Selection'),
    '#description' => t('Select the property and widget from your tawk.to account'),
  );
  $form['widget']['iframe'] = array(
    '#type' => 'markup',
    '#markup' => '<iframe id="tawkIframe" class="tawkto-property-widget-selection" src="" ></iframe>',
  );
  $form['widget']['page_id'] = array(
    '#type' => 'value',
    '#value' => $config->get('page_id'),
  }
```